### DISCUSSION TITLE
how to translate (in italian)?

### DISCUSSION THREAD

**Discussion by kintaro1981** [glpi-project/glpi-agentmonitor#7] [General]
I would like to translate it in italian is there some file to update?

In the mean time this are the string I see in my installation:

Info => Informazioni
Agent version => Versione Agente
Service status => Stato servizio
- Running => Avviato
- Paused => Sospeso
Startup type => Tipo di avvio
- Automatic => Automatico

Agent Status => Stato Agente
waiting => in attesa
running task Inventory => Attività inventario in corso
runnig task RemoteInventory => Attività inventario remoto in corso
running task Deploy => Attività distribuzione in corso
Force Inventory => Forza Inventario
Force inventory request successfully sent to GLPI Agent => La richiesta di inventario forzato è stata inviata correttamente all'Agente GLPI.
Close => Chiudi

**Comment by redddcyclone** [comment]
Hi @kintaro1981 

To add a new translation, you must edit the `GLPI-AgentMonitor.rc` file in Visual Studio (it's not a good idea to do it manually). Just open the "String Table" folder in Resource View, right-click any of the string tables and select "Insert Copy", choose the language from the dropdown list, click OK and open the newly created string table. Then, just edit the strings to your will and save the resource file after.

Then, after doing it, please open a PR in my repo [redddcyclone/glpi-agentmonitor](https://github.com/redddcyclone/glpi-agentmonitor) so we can merge the new translation.

Edit: also, the strings "waiting" and "running task (Task name)" (and every string displayed in the Agent Status box) are GLPI-Agent strings. They come from the /status page, so you won't be able to translate these in this project.

**Reply by kintaro1981** [reply]
I'm not used to Visual Studio, can I install the basic configuration of it or I need to install specific "modules"?

**Reply by redddcyclone** [reply]
You can install just VS 2022 Community with C++ development support (it will ask on the first startup or on installation). I think this will be enough to edit the resource file, compile the project and test the translation.

**Reply by kintaro1981** [reply]
done: https://github.com/redddcyclone/glpi-agentmonitor/pull/5

I tried with visual studio 2022 but it gave me a problem during .exe creation, it was detected as a virus: Gen:Variant.Tedy.375120 from BitDefender.

Can you check if is all ok?

**Reply by redddcyclone** [reply]
This is a false positive. It is commonly happening when compiling for x86. Could you try compiling for x64? If the same thing happens, put an exclusion for it in the antivirus if possible
